cdc/owner/feed_state_manager.go

```diff
@@ -370,6 +359,18 @@ func (m *feedStateManager) patchState(feedState model.FeedState) {
 			info.AdminJobType = adminJobType
 			changed = true
 		}
+		if updateEpoch {
+			previous := info.Epoch
+			ctx, cancel := context.WithTimeout(context.TODO(), 5*time.Second)
```

Comment:
Is `context.TODO()` ok here ?

Reply:
Tried, it will change a lot of unrelated code, I'll add a TODO here.